nixos/modules/services/web-servers/caddy.nix

```
@@ -68,6 +73,31 @@ in
       '';
     };
 
+    configs = mkOption {
+      type = types.attrsOf types.str;
```

```suggestion
      type = types.attrsOf types.lines;
```

`types.str` does not allow merging configuration, I suppose this is intended?

---

I didn't think there would be the need for merging between different configs, but this is a good point.